Makefile

```diff
@@ -156,16 +150,7 @@ build-plugin-admin-%: prepare-builder
 	fi
 
 	@echo build $(OS)-$(ARCH) plugin with version: $(BUILD_VERSION)
-	$(BUILDER) cli compile --version $(BUILD_VERSION) --ldflags "$(PLUGIN_LD_FLAGS)" --path ./cmd/plugin --artifacts "$(ARTIFACTS_ADMIN_DIR)/$(OS)/$(ARCH)/cli" --target ${OS}_${ARCH}
-
-
-.PHONY: publish-admin-plugins-all-local
-publish-admin-plugins-all-local: prepare-builder ## Publish CLI admin plugins locally for all supported os-arch
-	$(BUILDER) publish --type local --plugins "$(ADMIN_PLUGINS)" --version $(BUILD_VERSION) --os-arch "${ENVS}" --local-output-discovery-dir "$(TANZU_PLUGIN_PUBLISH_PATH)/discovery/admin" --local-output-distribution-dir "$(TANZU_PLUGIN_PUBLISH_PATH)/distribution" --input-artifact-dir $(ARTIFACTS_ADMIN_DIR)
-
-.PHONY: publish-admin-plugins-local
-publish-admin-plugins-local: prepare-builder ## Publish CLI admin plugins locally for current host os-arch only
-	$(BUILDER) publish --type local --plugins "$(ADMIN_PLUGINS)" --version $(BUILD_VERSION) --os-arch "${GOHOSTOS}-${GOHOSTARCH}" --local-output-discovery-dir "$(TANZU_PLUGIN_PUBLISH_PATH)/discovery/admin" --local-output-distribution-dir "$(TANZU_PLUGIN_PUBLISH_PATH)/distribution" --input-artifact-dir $(ARTIFACTS_ADMIN_DIR)
+	$(BUILDER) plugin build --version $(BUILD_VERSION) --path ./cmd/plugin --artifacts "$(ARTIFACTS_ADMIN_DIR)" --os-arch ${OS}_${ARCH}
```

Comment:
do we need to retain the --ldflags passing?

Reply:
Currently, plugins in this repository doesn't need `--ldflags` and hence I have removed it. In #80 I am adding generic tooling to build any plugin with the `plugin-tooling.mk` file where I have added the `--ldflags`.

Reply:
I do see it in #80 now, so effectively we are getting it back for plugins in this repo as well after 80 merges, so I am okay.
They are not used by default, but being able to used them is still useful, e.g. when building the plugins differently for debugging purposes.